api/resolvers/wallet.js

```diff
@@ -487,10 +487,10 @@ const resolvers = {
     },
     createWithdrawl: createWithdrawal,
     sendToLnAddr,
-    cancelInvoice: async (parent, { hash, hmac }, { models, lnd, boss }) => {
+    cancelInvoice: async (parent, { hash, hmac, userCancel }, { models, lnd, boss }) => {
       verifyHmac(hash, hmac)
       await finalizeHodlInvoice({ data: { hash }, lnd, models, boss })
-      return await models.invoice.findFirst({ where: { hash } })
+      return await models.invoice.update({ where: { hash }, data: { userCancel: !!userCancel } })
```

Comment:
we usually use `Number(x)` or `String(x)` so maybe I should have also used `Boolean(x)` here instead of `!!`.